src/classes/Pantry.js

```diff
@@ -28,8 +28,8 @@ class Pantry {
         null,
         ingredient.amount
       );
-    });
-    return ingredientList;
+    }).sort((a, b) => a.name - b.name)
+    return ingredientList
```

Comment:
We will have to remember to work on getting the names to sort during the refactor. Ill create an issue for it